### PR TITLE
Add comment stats on GitHub issues & pull requests

### PR DESCRIPTION
**Pull request by sandrobonazzola**
Fixes: #128
Signed-off-by: Sandro Bonazzola <sbonazzo@redhat.com>

### CODE REVIEW

**Review by kwk — CHANGES_REQUESTED** [review]
Can you please add a test for this?

**Comment by kwk** [comment]
Thank you @sandrobonazzola for coming up with this PR. I think this will be a great addition to `did`.

**Comment by sandrobonazzola** [comment]
> Can you please add a test for this?

Done

File: did/plugins/github.py [hunk 1]
```diff
@@ -154,6 +154,18 @@ def fetch(self):
             Issue(issue) for issue in self.parent.github.search(query)]
 
 
+class IssueCommented(Stats):
+    """ Issues commented """
+
+    def fetch(self):
+        log.info("Searching for issues commented by {0}".format(self.user))
```
Comment (kwk) [inline did/plugins/github.py:161]:
```suggestion
        log.info("Searching for issues commented on by {0}".format(self.user))
```

File: did/plugins/github.py [hunk 2]
```diff
@@ -167,6 +179,19 @@ def fetch(self):
             Issue(issue) for issue in self.parent.github.search(query)]
 
 
+class PullRequestsCommented(Stats):
+    """ Pull requests commented """
+
+    def fetch(self):
+        log.info("Searching for pull requests commented by {0}".format(
```
Comment (kwk) [inline did/plugins/github.py:186]:
```suggestion
        log.info("Searching for pull requests commented on by {0}".format(
```

**Review by kwk — CHANGES_REQUESTED** [review]
Thank you @sandrobonazzola for this contribution and for adding the tests. I've proposed a small log message change but then I think this PR is good to be merged. I ran tests locally and all passed. @psss do you have anything against me merging this?

**Comment by sandrobonazzola** [comment]
Log message fixes are ok for me

**Review by kwk — APPROVED** [review]
LGTM

**Review by psss — APPROVED** [review]
Thanks much for this improvement! Looks nice. I've just updated the commit message a bit to be aligned with the [recommendations](https://did.readthedocs.io/en/latest/contribute.html#commits).

**Pull request merged**